Gemfile.lock

```diff
@@ -366,4 +366,4 @@ RUBY VERSION
    ruby 2.2.5p319
 
 BUNDLED WITH
-   1.13.1
+   1.14.3
```

Comment:
Hmm i'm not sure why this got updated, but do you think you can downgrade your bundler to 1.13.1 and try to bundle install again? Might remove this diff